WordPress-Extra/ruleset.xml

```diff
@@ -16,6 +15,11 @@
 	<rule ref="Generic.CodeAnalysis.UselessOverridingMethod"/>
 	<rule ref="Generic.Classes.DuplicateClassName"/>
 	<rule ref="Generic.Strings.UnnecessaryStringConcat"/>
+	<rule ref="Squiz.PHP.NonExecutableCode"/>
+	<rule ref="Squiz.Operators.IncrementDecrementUsage"/>
+	<rule ref="Squiz.Operators.ValidLogicalOperators"/>
+	<rule ref="Squiz.PHP.DisallowComparisonAssignment"/>
```

Comment:
Sorry for the silence. Finally had a chance to look at this in properly.

By the looks of it, it does (forbid assignments using ternary conditions), though it is more specifically intended to catch things like `$var = ($foo === $bar)`.
See the unit tests.
https://github.com/squizlabs/PHP_CodeSniffer/blob/master/CodeSniffer/Standards/Squiz/Tests/PHP/DisallowComparisonAssignmentUnitTest.inc
https://github.com/squizlabs/PHP_CodeSniffer/blob/master/CodeSniffer/Standards/Squiz/Tests/PHP/DisallowComparisonAssignmentUnitTest.php

Would it be an idea to lower the error level for this sniff from `error` to `warning` ?